docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-nginx.asciidoc

```diff
@@ -6,10 +6,96 @@
 ++++
 
 // tag::nginx-jobs[]
-These {anomaly-job} wizards appear in {kib} if you use {filebeat} to ship access 
-logs from your http://nginx.org/[Nginx] HTTP servers to {es} and store it using 
-fields and datatypes from the Elastic Common Schema (ECS). For more details, see
-the {dfeed} and job definitions in
+
+These {anomaly-job} wizards appear in {kib} if you use either {filebeat} or the
+Nginx integration in {fleet} to ship access logs from your
+http://nginx.org/[Nginx] HTTP servers to {es}. The jobs assume that you use
+fields and data types from the Elastic Common Schema (ECS).
+
+[[nginx-access-logs]]
+== Nginx access logs
+
+These {anomaly-jobs} find unusual activity in HTTP access logs.
+
+For more details, see the {dfeed} and job definitions in
+https://github.com/elastic/integrations/blob/{branch}/packages/nginx/kibana/ml_module/nginx-Logs-ml.json[GitHub].
+Note that these jobs are available in {kib} only if data exists that matches the
+{dfeed} query.
+
+Job details:::
+
+Required {beats} or {agent} integrations:::
+
+* Nginx integration 
```

Comment:
I think it's either a copy/paste error or a module name and job detail description are missing. I assume it's the former.
```suggestion
```

Reply:
Good catch!  I'll double-check what went wrong there.